pyreal/explainers/gfi/shap_feature_importance.py

```diff
@@ -63,9 +63,9 @@ def get_importance(self):
             raise AttributeError("Instance has no explainer. Must call "
                                  "fit() before "
                                  "produce()")
-        x_explain = self.transform_to_x_model(self.x_train_orig)
-        x_explain_np = np.asanyarray(x_explain)
-        shap_values = np.array(self.explainer.shap_values(x_explain_np))
+        x_model = self.transform_to_x_model(self.x_train_orig)
```

Comment:
With the way this is currently written, x_algorithm must equal x_model, so really either could work. I changed this because I think x_model is more precise right now. In a future PR, we may update this so it can take the model with some transformers, in which cause x_algorithm would be accurate, but for now we can emphasize that this must be just the model ready data.